pkg/config/config.go

```diff
@@ -30,6 +30,8 @@ type Feed struct {
 	CronSchedule string `toml:"cron_schedule"`
 	// Quality to use for this feed
 	Quality model.Quality `toml:"quality"`
+	// Quality to use for this feed
+	CoverArtQuality model.Quality `toml:"cover_art_quality"`
```

Comment:
Could you please move this to `Custom` struct, where all customizations are happening.